app/commands/mentor/discussion/create.rb

```diff
@@ -18,6 +18,8 @@ def call
           # whole load need to fail.
           request.fulfilled!
 
+          request.locks.destroy_all
```

Comment:
```suggestion
          request.locks.where(mentor: mentor).destroy_all
```

Just in case two people have somehow managed to lock the same discussion?

Reply:
👍